pkgs/development/tools/vagrant/default.nix

```diff
@@ -96,11 +100,6 @@ stdenv.mkDerivation rec {
     cp -r opt "$out"
     cp -r usr/bin "$out"
     wrapProgram "$out/bin/vagrant" --prefix LD_LIBRARY_PATH : "$out/opt/vagrant/embedded/lib"
-
-    substituteInPlace $out/opt/vagrant/embedded/lib/ruby/2.2.0/x86_64-linux/rbconfig.rb \
-      --replace '"/bin/mkdir' '"${coreutils}/bin/mkdir'
-    substituteInPlace $out/opt/vagrant/embedded/lib/ruby/2.2.0/x86_64-linux/rbconfig.rb \
-      --replace "'/usr/bin/install" "'${coreutils}/bin/install"
```

Comment:
Hmm, my time is very limited and I have few more important things for me in nixpkgs than vagrant right now :/